tensorflow_addons/image/filters_test.py

```diff
@@ -18,74 +18,67 @@
 from __future__ import print_function
 
 import tensorflow as tf
-import median_filter_2d as md
+from tensorflow_addons.image import median_filter2d
 from tensorflow_addons.utils import test_utils
 
 
 @test_utils.run_all_in_graph_and_eager_modes
-class Median2DTest(tf.test.TestCase):
-    def _validateMedian_2d(self, inputs, expected_values, filter_shape=(3, 3)):
-
-        values_op = md.median_filter_2D(inputs)
-        with self.test_session(use_gpu=False) as sess:
-            if tf.executing_eagerly():
-                expected_values = expected_values.numpy()
-                values = values_op.numpy()
-            else:
-                expected_values = expected_values.eval()
-                values = values_op.eval()
-            self.assertShapeEqual(values, inputs)
-            self.assertShapeEqual(expected_values, values_op)
-            self.assertAllClose(expected_values, values)
-
-    def testfiltertuple(self):
+class MedianFilter2dTest(tf.test.TestCase):
+    def _validate_median_filter2d(self,
+                                  inputs,
+                                  expected_values,
+                                  filter_shape=(3, 3)):
+        output = median_filter2d(inputs, filter_shape=filter_shape)
+        self.assertAllClose(output, expected_values)
+
+    def test_filter_tuple(self):
         tf_img = tf.zeros([3, 4, 3], tf.int32)
 
         with self.assertRaisesRegexp(TypeError,
                                      'Filter shape must be a tuple'):
-            md.median_filter_2D(tf_img, 3)
-            md.median_filter_2D(tf_img, 3.5)
-            md.median_filter_2D(tf_img, 'dt')
-            md.median_filter_2D(tf_img, None)
+            median_filter2d(tf_img, 3)
+            median_filter2d(tf_img, 3.5)
+            median_filter2d(tf_img, 'dt')
+            median_filter2d(tf_img, None)
```

Comment:
Just found this is not robust. As long as one exception is raised within the context manager, the test will pass.